### PR TITLE
Zadanie 2 - Ola Barycka

### PR DESCRIPTION
**Pull request by olabarycka**

### CODE REVIEW

**Review by fidok15 — APPROVED** [review]
- brak wyświetlenia pełnych danych tabeli 
- do reprezentacji danych tytułów jak i pokładu użyłbym label encoding zamiast one-hot bo robi się przepastna ilość kolumn 
- do pokazania lepiej użyć  <a href='https://scikit-learn.org/stable/modules/generated/sklearn.metrics.classification_report.html'>classification_report</href> i <a href='https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html'>confusion_matrix</href>
- dobrze że jest zobaczenie wyników DummyClassifiera dla dwóch typów most_frequent i stratified
- przy badaniu parametrów osobiście przeszukałbym wszystkie w zakresię, a nie co pewną wartość szeczególnie przy min_splits i z min_splits działał na węższej wartościach 
- dobry dodatek, że jest wizualizacja modelu decyzyjnego drzewa 
- trafne wnioski

**Review by M4jonez — COMMENTED** [review]
1.Z podsumowania EDA widać bardzo dużo informacji wyciągniętych z danych, bardzo pomysłowo i dogłębnie
2.Fajnie wykorzystane funkcje pomocnicze, dodają czytelności kodu
3.Wizualizacje bardzo dobre

**Review by nuklearbanana — COMMENTED** [review]
Super wizualizacje z których można dużo wyciągnąć. 
Fajne wnioski, możesz sobie sprawdzić model RandomForest, bardziej odporny na przeuczenie, zwraca dokładniejsze predykcje i jest odporniejszy na szum w danych czego brakuje drzewu decyzyjnemu.
